scripts/pythonContractConverter.py

```diff
@@ -0,0 +1,115 @@
+# assumes all return types are strings
+def parseContract(file, funcName):
+    import re
+
+    fin = -1
+    try:
+        fin = open(file)
+    except:
+        print("[ERROR] {fname} cannot be opened".format(fname = file))
+        return ""
+
+    data = fin.read()
+    fin.close()
+    pattern = re.compile(r'#.*?\n')
+    data = re.sub(pattern, '', data)
+    pattern = re.compile(r"'''.*?'''", re.DOTALL)
+    data = re.sub(pattern, '', data)
+    if(("def " + funcName + "(") not in data):
+        print("[ERROR] {fname} does not contain a definition for \"{func}\"".format(fname = file, func = funcName))
+        return ""
```

Comment:
f-strings are useful here
```suggestion
    if f"def {funcName}(" not in data:
        print(f"[ERROR] {file} does not contain a definition for \"{funcName}\"")
        return ""
```